ops/build_and_package.sh

```diff
@@ -47,12 +47,6 @@ echo "Building bcdaworker..."
 go build
 echo "Packaging bcdaworker binary into RPM..."
 fpm -v $VERSION -s dir -t rpm -n bcdaworker bcdaworker=/usr/local/bin/bcdaworker
-cd ../ssas
```

Comment:
This file will be modified as part of BCDA-2018.  This is fine in here for now, but ultimately we'll probably see some merge conflicts once BCDA-2018 is merged to `master` and you'll want to take from that as the "source of truth".